CHANGELOG.md

```diff
@@ -7,6 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## Unreleased
 
+### Changed
+- Make Starks without constraints provable [#1552](https://github.com/0xPolygonZero/plonky2/pull/1552)
```

Comment:
Just to match previous format.
```suggestion
- Make Starks without constraints provable ([#1552](https://github.com/0xPolygonZero/plonky2/pull/1552))
```

Reply:
You can still prove CTLs/lookups (which are some kind of fixed constraints though)

Reply:
I think the proof of knowledge property appears once you add the OOD evaluation. But I'm not sure this will be the case if you don't have CTLs or lookups.

Reply:
> You can still prove CTLs/lookups

Yeah though you don't necessarily have them. But I get your point.

Reply:
It works without CTLs/lookups out-of-the-box. I added an example of a completely unconstrained stark